<!--↓エラーメッセージ表示枠-->
<% if model_info.errors.any? %>
  <div id="error_explanation">
    <h2><%= get_tablename_by_model(model_info) %>
      情報登tt録で<%= pluralize(model_info.errors.count, "個") %> のエラーが発生しています。</h2>
    <ul>
    <% model_info.errors.full_messages.each do |message| %>
      <li><%= message %></li>
    <% end %>
    </ul>
  </div>
<% end %>
